package.json

```diff
@@ -10,20 +10,20 @@
         "@vitejs/plugin-vue": "5.2.1",
         "autoprefixer": "10.4.20",
         "axios": "1.7.9",
-        "laravel-echo": "1.17.1",
-        "laravel-vite-plugin": "1.1.1",
-        "postcss": "8.4.49",
-        "pusher-js": "8.4.0-rc2",
+        "laravel-echo": "2.0.2",
+        "laravel-vite-plugin": "^1.2.0",
```

Comment:
I'd recommend having fixed versions everywhere. Trust me, this prevents weird behaviour sometimes.